lib/svg-sprite/mode/base.js

```diff
@@ -135,7 +135,7 @@ SVGSpriteBase.prototype._initData = function(data) {
                 renderConfig.dest = path.resolve(this.config.dest, this.config.example.dest);
             }
         } else if (this.config.example !== true) {
-            renderConfig = false;
+            renderConfig = false; // todo: fix bug with this branch on line 142 (boolean has no "dest" property)
```

Comment:
Which branch are you referring to?

Reply:
@XhmikosR current `else if` branch (line 138)

Reply:
Ah, I see. Indeed, good catch 👍